__mocks__/snapshotResolver.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable */
+const path = require('path');
```

Comment:
I don't see path used in this file? Is this a weird jest thing?

Reply:
yeah, path is used to mock node's native lib b/c those aren't mocked by default https://jestjs.io/docs/en/manual-mocks